### PR TITLE
Add joint velocity limits task

### PR DESCRIPTION
**Pull request by davidegorbani**
With this PR I add a task in the IK to set the joint velocity limits.

CC @Gianlucamilani @claudia-lat

### CODE REVIEW

File: src/IK/src/JointVelocityLimitsTask.cpp [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        || !ptr->getParameter("lower_limits", m_lowerLimits))
+    {
+        log()->error("{} Error while retrieving the 'upper_limits' and/or the 'lower_limit' "
+                     "parameters.",
+                     errorPrefix);
+        return false;
+    }
+
+    // set the description
+    m_description = "Joint limits task";
```
Comment (GiulioRomualdi) [inline src/IK/src/JointVelocityLimitsTask.cpp:128]:
```suggestion
    m_description = "Joint velocity limits task";
```

**Review by GiulioRomualdi — COMMENTED** [review]
Thank you! Can you add the pybind11 bindings?

**Comment by GiulioRomualdi** [comment]
The CI fails with 

```
       joint_velocity_limits_task = blf.ik.JointVelocityLimitsTask()
E       AttributeError: module 'bipedal_locomotion_framework.bindings.ik' has no attribute 'JointVelocityLimitsTask'
```

Let me check why

**Comment by GiulioRomualdi** [comment]
Got it. You should call the function `CreateJointVelocityLimitsTask` here

https://github.com/davidegorbani/bipedal-locomotion-framework/blob/fe4c89b37186208c3d91413a601f5094f644bcf4/bindings/python/IK/src/Module.cpp#L43

**Comment by GiulioRomualdi** [comment]
Done in https://github.com/ami-iit/bipedal-locomotion-framework/pull/879/commits/3bd8fbcabe390d999d4917438ed91b8c0127016c

**Comment by davidegorbani** [comment]
> Done in [3bd8fbc](https://github.com/ami-iit/bipedal-locomotion-framework/commit/3bd8fbcabe390d999d4917438ed91b8c0127016c)

Thank you!

**Review by GiulioRomualdi — APPROVED** [review]

**Pull request merged**